js/util/worker_pool.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var assert = require('assert');
+var WebWorker = require('./web_worker');
+var browser = require('./browser');
+
+module.exports = WorkerPool;
+
+WorkerPool.WORKER_COUNT = Math.max(browser.hardwareConcurrency - 1, 1);
```

Comment:
`WorkerPool.WORKER_COUNT` -> `WorkerPool.workerCount`

I'd like to start removing `MACRO_CASE` identifiers from the codebase. Javascript doesn't have any macros. The property's scope is already implied by its context. 

> Underscores with upper case, as in `UPPER_CASE`, are commonly used for C preprocessor macros, hence known as `MACRO_CASE`, and for environment variables in Unix, such as `BASH_VERSION` in bash.